markdown/get-started-basics/get-started-basics.md

```diff
@@ -0,0 +1,394 @@
+author: Aaron Lee
+summary: Absolute basics of first connecting to a Solace event broker
+id: get-started-basics
+tags: howto
+categories:
+environments: Web
+status: Published
+feedback link: https://solace.community/
+analytics account: UA-3921398-10
+
+# Absolute Solace Basics - Getting Started with Solace PubSub+ Event Broker
+
+
+
+
+
+
+## What you'll learn: Overview
+
+Duration: 0:01:00
+
+In this introductory CodeLab, you'll learn how to:
+
+* Setup a **Solace PubSub+ event broker**, and/or start a **PubSub+ Cloud** service
+* Login to the PubSub+ Manager GUI
+* Utilize test WebSockets applications to send and receive data using **Publish-Subscribe**, the fundamental communication pattern for event-driven applications
+* Improve routing and filtering using **topic hierarchies and wildcards**
+* Create a durable queue for **Guaranteed** Messaging
+* Test Guaranteed messaging to ensure no message loss
+* Enable **Replay** on the Solace broker, and test it
+
+![picture](img/broker-img.png)
+
+
+Positive
+: This CodeLab does not cover the Event Portal. That will be addressed in a separate CodeLab.
+
+
+
+
+
+
+
+## Prerequisites
+
+Duration: 0:01:00
+
+In order to complete this CodeLab, you will need to get access to a Solace PubSub+ event broker.
+
+At minimum, you will need to have internet access and a browser to use Solace PubSub+ Cloud.
+
+To run the software broker, you'll need to have Docker, VirtualBox, VMWare Player, or another similar technology available to you.  More details in the next step.
+
+
+
+
+
+
+
+## Starting a Solace PubSub+ Event Broker
+
+Duration: 0:10:00
+
+Let's get access to a Solace PubSub+ event broker!
+
+### Software Broker
+
+There are multiple options on how to install/download/configure/run the PubSub+ broker:
+
+* If you Docker installed, this is [probably the easist way](https://solace.com/products/event-broker/software/getting-started/). Simply copy this line into a terminal to pull the latest image from Docker Hub, and boot it:
+
+```
+docker run -d -p 8080:8080 -p 55555:55555 -p:8008:8008 -p:1883:1883 -p:8000:8000 -p:5672:5672 -p:9000:9000 -p:2222:2222 --shm-size=2g --env username_admin_globalaccesslevel=admin --env username_admin_password=admin --name=solace solace/solace-pubsub-standard
```

Comment:
Let's map 55555 to 4444 as 55555 no longer works on Mac

Reply:
Boooo Mac!!!

Reply:
I'll add a comment specifically for Mac people.  I don't think Windows and Linux people should have to change their default Solace ports because of OSX.  😬